Dockerfile

```
@@ -12,7 +12,7 @@ EXPOSE ${PORT}
 
 COPY .docker/run.sh ./
 
-ARG CONFIG_FILE=tests/test_config.json
+ARG CONFIG_FILE
```

I would suggest to return this line to its original state, or maybe instead to something like:
```suggestion
ARG CONFIG_FILE=.docker/config.json
```
The value `config.json` here is simply a fallback, if `CONFIG_FILE` has not been specified in `docker-compose.yml`.

---

Do we want the fallback to be something that points to a fake url "gh_actions_host" though? Could revert it to the test config which has localhost instead

---

Since an implementer would need to either change this or the `docker-compose.yml` file in any case, this shouldn't be an issue.
They would have to change the actual config file in the end as well. So leaving it to point to `.docker/config.json` seems fine. I'd mucher rather have that than a test config, which has nothing to do with docker.